### DISCUSSION TITLE
How to configure mysql connection string from Azure web app

### DISCUSSION THREAD

**Discussion by ruixue8080** [OrchardCMS/OrchardCore#15867] [Q&A]
Our project is deployed in Azure web app, and we have different connection string for production and testing environment. Can I configure mysql connection string for Orchard from Azure web app configration? what key should I use if yes.

according to this article: https://docs.orchardcore.net/en/dev/docs/reference/core/Configuration/, azure web app has the highest priority for configure database connection string, however my configuration in web app just not work.

It only work if I set mysql connection string in AppData/Site/{MySite}/appsettings.json, it NOT work if I set connection string from {root of startup project}/appsettings.json, neither not work for web app configuration.

**Comment by deanmarcussen** [comment]
If you want to use environment variables from azure web app configuration here is a working example

`OrchardCore__Default__ConnectionString = connectionstringvalue` 

where `Default` is the name of the default shell. Replace with a tenant name if you are using tenants.

I don't quite understand what you have tried with `appsettings.json` so please post your code if that is how you are trying to configure it, and it is not working.
